examples/sign-send-tx-quai.js

```diff
@@ -0,0 +1,39 @@
+const quais = require('../lib/commonjs/quais');
+require('dotenv').config();
+
+async function main() {
+	
+	// create provider
+	const provider = new quais.JsonRpcProvider(process.env.RPC_URL);
+	
+	// create wallet
+	const mnemonic = quais.Mnemonic.fromPhrase(process.env.MNEMONIC);
+	const quaiWallet = quais.QuaiHDWallet.fromMnemonic(mnemonic);
+	quaiWallet.connect(provider);
+
+	// Create tx
+	const addressInfo1 = await quaiWallet.getNextAddress(0, quais.Zone.Cyprus1);
+	const from = addressInfo1.address;
+	const txObj = new quais.QuaiTransaction(from);
+	txObj.chainId = BigInt(9000);
+	txObj.nonce = await provider.getTransactionCount(from, 'latest');
+	txObj.gasLimit = BigInt(1000000);
+	txObj.maxPriorityFeePerGas = BigInt(10000000000),
+	txObj.maxFeePerGas = BigInt(30000000000000),
```

Comment:
I think we should either remove `chainId`, `nonce`, `gasLimit`, `maxFeePerGas` and `maxPriorityFeePerGas` OR we should specify in some comments that these fields will be defined for the user if not provided.